package.json

```diff
@@ -40,7 +40,7 @@
     "bignumber.js": "^4.0.1",
     "eslint": "^4.3.0",
     "eslint-config-google": "^0.9.1",
-    "ethereumjs-abi": "^0.6.4",
+    "ethereumjs-abi": "github:yondonfu/ethereumjs-abi#abi-offset-fixed-arrays",
```

Comment:
There is an ABI encoding bug in the ethereumjs-abi package - I've opened a PR, but until that is merged in this will install the branch with the fix on my fork